src/models/line.js

```diff
@@ -121,8 +123,8 @@ nv.models.line = function() {
                         .interpolate(interpolate)
                         .defined(defined)
                         .x(function(d,i) { return nv.utils.NaNtoZero(x0(getX(d,i))) })
-                        .y0(function(d,i) { return nv.utils.NaNtoZero(y0(getY(d,i))) })
-                        .y1(function(d,i) { return y0( y.domain()[0] <= 0 ? y.domain()[1] >= 0 ? 0 : y.domain()[1] : y.domain()[0] ) })
+                        .y0(function(d,i) { return y0(getYLower(d,i)).length == 0 ? nv.utils.NaNtoZero(y(getY(d,i))) :  y0(getYLower(d,i))})
+                        .y1(function(d,i) { return y0(getYUpper(d,i)).length == 0 ? y( y.domain()[0] <= 0 ? y.domain()[1] >= 0 ? 0 : y.domain()[1] : y.domain()[0] ) : y0(getYUpper(d,i))})
```

Comment:
I just happened to notice that within the main .y0() you changed it to call y() instead of y0() if getYLower isn't defined, and within the main .y1() you changed it to call y0() instead of y() if getYLower isn't defined...  is that supposed to be correct?